src/Compilers/CSharp/Test/CommandLine/CommandLineTests.cs

```diff
@@ -9811,7 +9811,7 @@ private static int OccurrenceCount(string source, string word)
 
             if (expectedInfoCount == 0)
             {
-                Assert.DoesNotContain("info", output, StringComparison.Ordinal);
+                Assert.DoesNotContain("info", output.Split('\n').First(), StringComparison.Ordinal);
```

Comment:
yes.  the problem was that you might get something like a stack-trace in a compiler message (i.e. when analyzers crash we include their stack in the message).  But we don't want to search that stack for things like 'info/warn/error' because the stack trace message may contain that.  Instead, we just check the first line of the compiler diagnostic message as that is the part where the compiler itself prints out if this is a warning/info/error.

note: this is a major hack in teh first place as all these tests run the compiler then grok the console output.  this just makes us resilient to that problem (which i was hitting).